File: EODS.vb
Public Function XML()

    // declare and build String containing XML
    Dim xmlString As String
    xmlString = "<KnowWho><Customer> ... </Customer><Address> ... </Address></KnowWho>";

    // declare new instance of the service
    Dim serviceInstance As New Namespace.ReferenceName.ElectedOfficialsDirectoryService()

    // call the appropriate function, passing the String
    Dim returnedXml As String
    returnedXml = serviceInstance.GetService(xmlString)

    // if all was successful, returnedXml now contains a string representation of the returned XML

End Function

Public Function JSON()

    // declare and build String containing JSON
    Dim jsonString As String
    jsonString = "{ KnowWho: { Customer: { ... }, Address: { ... } } }";

    // declare new instance of the service
    Dim serviceInstance As New Namespace.ReferenceName.ElectedOfficialsDirectoryService()

    // call the appropriate function, passing the String
    Dim returnedJson As String
    returnedJson = serviceInstance.GetService(jsonString)

    // if all was successful, returnedJson now contains a string representation of the returned JSON

End Function

File: EODS.cs
public void XML()
{
    string content = "";
    string inputXml = "<KnowWho><Customer>...</Customer><Address>...</Address></KnowWho>";
            
    ElectedOfficialsDirectoryService.ElectedOfficialsDirectoryService srvce = new ElectedOfficialsDirectoryService.ElectedOfficialsDirectoryService();
    try
    {
        content = srvce.GetService(inputXml);
    }
    catch (Exception ex)
    {
        Console.WriteLine(ex.Message);
    }
}

public void JSON()
{
    string content = "";
    string inputJson = "{ KnowWho: { Customer: { ... }, Address: { ... } } }";
            
    ElectedOfficialsDirectoryService.ElectedOfficialsDirectoryService srvce = new ElectedOfficialsDirectoryService.ElectedOfficialsDirectoryService();
    try
    {
        content = srvce.GetService(inputJson);
    }
    catch (Exception ex)
    {
        Console.WriteLine(ex.Message);
    }
}